mautic/mautic
# TPROD-282 - Controller reference syntax updated to Symfony 5

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [x]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | https://mautic.atlassian.net/browse/TPROD-282 



#### Description:


This PR rewrites all controller references from `bundle:controller:action` syntax to new `serviceOrFqcn::method` syntax. The `bundle:controller:action` syntax will be no longer supported in Symfony 5.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Make sure everything is working as all controller references have been rewritten.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11055"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
